src/script/pages/app-publish.ts

```diff
@@ -0,0 +1,142 @@
+import { css, customElement, html, LitElement } from 'lit-element';
+import '../components/app-header';
+import '../components/app-sidebar';
+import { testManifest } from '../services/tests/manifest';
+import { testServiceWorker } from '../services/tests/service-worker';
+@customElement('app-publish')
+export class AppPublish extends LitElement {
+  constructor() {
+    super();
+  }
+
+  async firstUpdated() {
+    const search = new URLSearchParams(location.search);
+    const site = search.get('site');
+
+    if (site) {
+      const manifestTestresults = await testManifest(site);
```

Comment:
So these are probably just in here from where you copy pasted the other page to use as a template, but would you mind removing these calls? They are not needed for the publish page.

Reply:
Got it I thought they have something to do with the pwd